apps/dialtone-documentation/docs/components/scroller.md

```diff
@@ -97,7 +97,17 @@ storybook: https://dialtone.dialpad.com/vue/vue3/?path=/story/components-scrolle
 </dt-scroller>
 </code-well-header>
 
-```html
+<code-example-tabs
+htmlCode='
+<div data-v-f28a4c1a="" class="vue-recycle-scroller ready direction-vertical scroller" tabindex="0" style="width: 300px; height: 200px;">
+  <div class="vue-recycle-scroller__item-wrapper" style="min-height: 1600px;">
+    <div class="vue-recycle-scroller__item-view" style="transform: translateY(0px) translateX(0px);"><div data-v-f28a4c1a="" class="user">User 0</div></div>
+    <div class="vue-recycle-scroller__item-view" style="transform: translateY(32px) translateX(0px);"><div data-v-f28a4c1a="" class="user">User 1</div></div>
+   ...
+  </div>
+</div>
+'
```

Comment:
Hmm I'm wondering if we should just not show HTML at all for this component since it's "non visual"? I don't really think this is useful to anyone, and probably causes more confusion than anything else.

Reply:
It still shows you how it is being structured through the HTML, dunno, I dont see it wrong to have it, vue is the first tab showing in the example so.. If you want to see how it sees in HTML, its a second option.

Reply:
I will remove it, it does not adding any value